labgrid/driver/usbstoragedriver.py

```diff
@@ -122,9 +122,9 @@ def write_image(self, filename=None, mode=Mode.DD, partition=None, skip=0, seek=
     @Driver.check_active
     @step(result=True)
     def get_size(self):
-        args = ["cat", "/sys/class/block/{}/size".format(self.storage.path[5:])]
+        args = ["/sbin/blockdev", "--getsize64", self.storage.path]
```

Comment:
This shouldn't be `/sbin/blockdev`, it should rather be `blockdev`. `PATH` needs to be setup correctly in the user environment, not all distributions have `blockdev` in sbin.

Reply:
blockdev also behaves differently, instead of returning `0` it will return `blockdev: cannot open /dev/sdb: No medium found` and exit with `1`, this handling needs to be adapted here.

Reply:
You should also drop the `try .. except` from `write_image()`.